python/tvm/relay/frontend/paddlepaddle.py

```diff
@@ -191,57 +336,76 @@ def convert_dropout(g, op, block):
     """Operator converter for dropout."""
 
     x = g.get_node(op.input("X")[0])
-    out = _op.copy(x)
+    g.add_node(op.output("Out")[0], x)
+
+
+def convert_dot(g, op, block):
+    """Operator converter for dot."""
+
+    x = g.get_node(op.input("X")[0])
+    y = g.get_node(op.input("Y")[0])
+
+    out = _op.sum(_op.multiply(x, y), axis=[-1], keepdims=True)
```

Comment:
You might want to note the semantics of paddle paddle's dot operator. Namely how it also operates on 2d-vectors (and hence why axis=[-1]).

I have not seen this elsewhere

Reply:
`paddle.dot` : https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/dot_en.html

It's similar with `torch.dot`, while `torch.dot` only supports 1D tensor.  
In PaddlePaddle, inputs should be both 1D or 2D tensor. When it is 2d, the first dimension of this matrix is the batch dimension.

For clarify, I also put this explanation in code